src/io/show/MetabolicModel.jl

```diff
@@ -3,7 +3,7 @@ Pretty printing of everything metabolic-modelish.
 """
 function Base.show(io::IO, ::MIME"text/plain", m::MetabolicModel)
     _pretty_print_keyvals(io, "", "Metabolic model of type $(typeof(m))")
-    if prod(size(stoichiometry(m))) < 5_000_000
+    if prod(size(stoichiometry(m))) < _constants.default_stoichiometrix_view_size
```

Comment:
coult we have something more like `maximum(size(stoichiometry(m))) < 500`
Also, this kinda creates the matrix and then throws it away, might be better to call n_reactions() and n_metabolites()

Reply:
(also can we please have this in a separate PR?)